test/functional/home_controller_test.rb

```diff
@@ -11,7 +11,7 @@ def setup
 
     get :home
     assert_response :success
-    assert_select "title", Sanitize.clean('&#127880;') + (" Public Lab: #{title}")
+    assert_select "title", Sanitize.clean('&#127880;') + (" Public Lab: #{title}") # Test failing here
```

Comment:
This is giving a Nokogiri error, which I'm unable to debug.

```
ERROR HomeControllerTest#test_should_get_home (26.22s)
Minitest::UnexpectedError:         TypeError: wrong argument type Nokogiri::HTML5::DocumentFragment (expected Data)
            test/functional/home_controller_test.rb:14:in `block in <class:HomeControllerTest>'

```
Is this some version issue? The newer version experts some other type of data, and the tests haven't been updated to reflect that?

Reply:
Hmm, what version change are we making that could result in this? Maybe this will also just resolve because the home controller will render properly. But if not, that sanitized string is an emoji so maybe that's causing the problem. Let's try fixing the home controller first though.

Reply:
Resolved now!